owl/EASE-OBJ.owl

```diff
@@ -2595,6 +2615,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Size -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Size">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Intrinsic"/>
+        <rdfs:comment>Size is the magnitude or dimension of a thing. Size can be measured as length, width, height, diameter, perimeter, area, volume, or mass.</rdfs:comment>
```

Comment:
mass? I think we have a Mass Quality or? It would not be very intuitive to put it as sub-class of Size

Reply:
done.